.github/workflows/package-cloud-download-schedule.yml

```diff
@@ -18,7 +18,8 @@ jobs:
       fail-fast: false
       matrix:
         parallel_index: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
-        repo: [community, enterprise, community_nightlies, enterprise_nightlies]
+        
```

Comment:
Is this empty line necessary?

Reply:
I'm adding an empty line to seperate parameters and jobs. I think it is more readable